test/powershell/Language/Scripting/CheckRestrictedlanguage.Tests.ps1

```diff
@@ -0,0 +1,112 @@
+﻿#  <Test>
+#    <summary>Test restricted language check method on scriptblocks</summary>
+#  </Test>
+Describe "Test restricted language check method on scriptblocks" -Tags "CI" {
+        BeforeAll {
+            set-strictmode -v 2
+            function list {
+
+            $l = new-object system.collections.generic.list[string]
```

Comment:
fixed

---
In reply to: [76495042](https://github.com/powershell/powershell/pull/2086#discussion_r76495042) [](ancestors = 76495042)